gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
```

Comment:
Why did we update version of Gradle?)

Reply:
Why not, we skiped 6 versions....

Reply:
We started the project with an obsolete version
This is not critical at the moment, so why not